src/lib/datepicker/calendar.ts

```diff
@@ -29,7 +30,27 @@ import {MatDatepickerIntl} from './datepicker-intl';
 import {MatMonthView} from './month-view';
 import {MatMultiYearView, yearsPerPage} from './multi-year-view';
 import {MatYearView} from './year-view';
+import {ComponentPortal, ComponentType, Portal} from '@angular/cdk/portal';
 
+/** Default header for MatCalendar */
+@Component({
+    moduleId: module.id,
+    selector: 'mat-calendar-header',
+    templateUrl: 'calendar-header.html',
+    encapsulation: ViewEncapsulation.None,
+    preserveWhitespaces: false,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MatCalendarHeader {
+  constructor(@Host() public calendar: MatCalendar<any>,
+              private _intl: MatDatepickerIntl,
+              changeDetectorRef: ChangeDetectorRef) {
+
```

Comment:
nit: remove this blank line

Reply:
done in a88fecbec2c3292017613bf4e75c38ff7c632042